chain/network/src/peer_manager/tests/routing.rs

```diff
@@ -756,7 +760,7 @@ async fn blacklist_all() {
     tracing::info!(target:"test", "start two nodes with 0 blacklisting everything");
     let mut cfgs = make_configs(&chain, rng, 2, 2, true);
     cfgs[0].peer_store.blacklist =
-        [blacklist::Entry::from_ip(Ipv4Addr::LOCALHOST.into())].into_iter().collect();
+        [blacklist::Entry::from_ip(Ipv6Addr::LOCALHOST.into())].into_iter().collect();
```

Comment:
Wouldn’t this have worked either way, given that blacklist converts IPs to IPv6 anyway?

Reply:
Hm, that’s a little scary, but you know this code better so I’ll leave the call here up to you.